scrapers/scrape_bs.py

```diff
@@ -61,9 +80,9 @@
                     <p>Mit Stand Mittwoch, 1. April 2020, 10 Uhr, liegen insgesamt 691 positive F&auml;lle von Personen mit Wohnsitz im Kanton Basel-Stadt vor. 323 Personen der 691 positiv Getesteten und damit &uuml;ber 45 Prozent sind wieder genesen.</p>
 """
 
-# Use non-greedy matching.
-print('Date and time:', sc.find(r'Stand\s*[A-Za-z]*,?\s*(.+?),\s*(?:liegen\s*)?insgesamt', d))
-print('Confirmed cases:', sc.find(r'(?:insgesamt\s*)?([0-9]+)\s*positive', d))
-print('Recovered:', sc.find(r'([0-9]+) Personen der [0-9]+ positiv Getesteten .+ sind wieder genesen', d))
-print('Hospitalized:', sc.find(r'Aktuell befinden sich ([0-9]+) Einwohnerinnen und Einwohner des Kantons Basel-Stadt aufgrund einer Covid-19-Infektion in Spitalpflege', d))
-print('ICU:', sc.find(r'Insgesamt ([0-9]+) Personen benötigen Intensivpflege', d))
+print('Date and time:', sc.find(r'Stand\s*[A-Za-z]*,?\s*(.+?),\s*(?:liegen\s*)?insgesamt', text))
+print("Confirmed cases: ", result["NUMCUL_CONF_RESIDENTS"])
+print("Death: ", result["NUMCUL_DECEASED"])
+print("Recovered: ", result["NUMCUL_RELEASED"])
+print("Hospitalized: ", result["NUMCUL_HOSP"])
+print("ICU: ", result["NUMCUL_ICU"])
```

Comment:
Minor thing, but please remove the trailing spaces after a colon (print automatically puts a separator, default space, between printed elements), and use single quotes, to be consistent.

Example:

```python3
print('Confirmed cases:', result["NUMCUL_CONF_RESIDENTS"])
```
Same for other print statements here.

Thanks.